frame/dapps-staking/src/weights.rs

```diff
@@ -29,7 +31,7 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: DappsStaking RegisteredDapps (r:1 w:1)
 	// Storage: DappsStaking PreApprovalIsEnabled (r:1 w:0)
 	fn register() -> Weight {
-		(32_139_000 as Weight)
+		(60_000_000 as Weight)
```

Comment:
Please only modify the 3 benchmarks you impacted, let's leave the other numbers as they are for now.